packages/electron-updater/src/windowsExecutableCodeSignatureVerifier.ts

```diff
@@ -16,16 +16,16 @@ export function verifySignature(publisherNames: Array<string>, tempUpdateFile: s
       try {
         if (error != null || stderr) {
           handleError(logger, error, stderr)
-          resolve(null)
-          return
+          logger.warn(`Can't verify the signature: ${error}`);
+          throw error;
         }
 
         const data = parseOut(stdout)
         if (data.Status === 0) {
           const name = parseDn(data.SignerCertificate.Subject).get("CN")!
           if (publisherNames.includes(name)) {
-            resolve(null)
-            return
+            logger.warn(`Cannot execute due to unknown results: ${error}. Aborting update.`);
+            throw error;
```

Comment:
I don't think you want to throw an error here -- this is where verification actually succeeds. We've been shipping this [patch](https://github.com/iotaledger/trinity-wallet/blob/cf6e1099de10452a43cf59b144ff759788e77c72/src/desktop/patches/electron-updater%2B4.0.14.patch) with our app if you'd like to use that

Reply:
Hi @rajivshah3 :),
you're right, I confused 2 patches I was working on and while integrating the 2 I forgot to leave it as it was before, since as you explained previously it's the correct behavior :).

I'll provide the new PR as soon I can today :).

Regards and thanks,
Mik